src/Control/Session.php

```diff
@@ -321,28 +316,30 @@ public function start(HTTPRequest $request)
                 session_cache_limiter($limiter);
             }
 
-            session_start();
+            $sessionParameters = [
+                "cookie_path" => $path,
+                "cookie_domain" => $domain ?: "",
+                "cookie_secure" => $secure,
+                "cookie_httponly" => true
+            ];
+
+            if ($timeout) {
+                $sessionParameters["cookie_lifetime"] = $timeout;
+            }
+
+            session_start($sessionParameters);
```

Comment:
The `options` argument for `session_start` [has been introduced](http://php.net/manual/en/function.session-start.php) in `PHP 7.0`.
I believe we should keep compatibility with `PHP 5.6` for a while yet.

Reply:
I _think_ that providing these params in 5.6 is just a no-op - meaning that this wouldn't break 5.6 but the bug won't be fixed either.

Reply:
Yes, PHP allows passing extra/undeclared arguments into a function. What I meant is we should still keep using `session_set_cookie_params` for a while.